infra/cifuzz/actions/action.yml

```diff
@@ -9,9 +9,13 @@ inputs:
     description: 'The total time allotted for fuzzing in seconds.'
     required: true
     default: 360
+  dry-run:
+    description: 'If set, run the action without actually reporting a failure.'
+    default: False
```

Comment:
nit: for consistency with line 11, lower case the "false" ?

Reply:
Fixed.